apps/microtvm/reference-vm/zephyr/base-box/base_box_test.sh

```diff
@@ -37,3 +37,5 @@ if [ $board == "stm32f746xx" ]; then
 else
     pytest tests/micro/zephyr/test_zephyr_aot.py --zephyr-board=${board}
 fi
+
+pytest tests/micro/zephyr/test_zephyr_armv7m.py --zephyr-board=${board}
```

Comment:
could you move this to `tests/scripts/task_python_microtvm.sh`? This is just the base-box pre-release test, where we just want to do a smoke test to make sure that Zephyr is working with attached hardware.

Reply:
@areusch we merged a branch with modifications for another issue by mistake here. All these changes will be reverted, but we take all you comments into account and fix it in #8990